source/screens/caseScreens/CaseOverview.tsx

```diff
@@ -226,11 +226,12 @@ const computeCaseCardComponent = (
     cardProps.onClick = undefined;
   }
 
-  const shouldShowPin = isWaitingForSign && !isCoApplicant;
+  const shouldShowPin = !(isSigned || isRandomCheckRequired);
```

Comment:
This line change doesn't seem to work correctly, but if I test with reverting only this line then everything work as expected!